src/test/java/com/training/web/tests/flipkart/FlipkartTest.java

```diff
@@ -53,13 +64,18 @@ public void testPriceSort() throws WebDriverException, ParseException {
                 resultPage.clickNextPage();
             }
         }
+        getWebDriver().close();
```

Comment:
Why do we have this in test? shouldn't this be in after each in the base test - since this is a common code?